src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -41,6 +42,6 @@ public List<CinemaHallResponseDto> getAll() {
         return cinemaHallService.getAll()
                 .stream()
                 .map(cinemaHallResponseDtoMapper::mapToDto)
-                .toList();
+                .collect(Collectors.toList());
```

Comment:
can we use jdk 17 and `.toList`?

Reply:
Have problem with 17 jdk in this project